### PR TITLE
Adds Closed icon to differentiate Merged and Closed PRs -RE issue 142

### PR DESCRIPTION
**Pull request by David-Byrne**
# Fixes :shipit: :shipit: :shipit:
https://github.com/jenkoian/hacktoberfest-checker/issues/142

# Summary :1st_place_medal: :1st_place_medal: :1st_place_medal:
Probably needs some work. Not sure if the benefit added by the icon justifies the increase in complexity...

### CODE REVIEW

**Comment by jenkoian** [comment]
@David-Byrne this is looking good to me, let me know when you feel it's finished and I'll give it a more thorough review 👍

**Comment by David-Byrne** [comment]
Thanks @jenkoian, I've fixed the linting errors so it's ok from that point of view.

It increases the number of requests to the Github API by a fair amount though but we can't seem to get around that. Up to you whether the benefit justifies this extra cost. 
I'd also have liked to have it cleaner than counting the number of promises that have been fulfilled, but the API just returns errors if a change hasn't been merged so that stops us from using Promise.all()

File: controllers/index.js [hunk 1]
```diff
@@ -44,8 +44,41 @@ exports.index = (req, res) => {
 
                 prs.push(returnedEvent);
             });
+
+            const requests = [];
+            _.forEach(prs, pr => {
+                const [repoOwner, repoName] = pr.repo_name.split('/');
```
Comment (jenkoian) [inline controllers/index.js]: This errors for me locally, possibly due to the version of node I'm on, but I think we should support form node 6 and up. Are you able to change this?

Comment (David-Byrne) [reply controllers/index.js]: I thought destructuring was supported by version 6? See https://stackoverflow.com/questions/17379277/destructuring-in-node-js

I've no problem changing it though if needed

**Comment by jenkoian** [comment]
@David-Byrne I want to merge this but want to add some logging of rate limits, so I can keep an eye on it, I won't get chance to do that today, so was wondering if you were able to add that? The response from github should include the amount of requests left, then it should just be a case of calling console.log.

File: controllers/index.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ exports.index = (req, res) => {
         ]).then(gitData => {
             const foundPrs = gitData[0];
             const user     = gitData[1];
+            console.log("API calls remaining: " + user.meta['x-ratelimit-remaining']);
```
Comment (jenkoian) [inline controllers/index.js]: Needs to be single quotes for the build :)

**Comment by jenkoian** [comment]
Thank you @David-Byrne absolutely brilliant PR!

**Pull request merged**